spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -340,6 +351,28 @@ def within_summary_section
     end
   end
 
+  def and_I_can_see_a_steps_overview_section
+    within('.govuk-table.step-list') do
+      expect(page).to have_content('Step Title Actions')
+      expect(page).to have_content('Check how awesome you are')
+      expect(page).to have_content('Dress like the Fonz')
+      expect(page).to have_css('.govuk-table__body .govuk-table__row', :count => 2)
+    end
+  end
+
+  def and_I_can_see_an_empty_steps_overview_section
+    expect(page).not_to have_css('.govuk-table.step-list')
+    expect(page).to have_content('No steps have been added yet.')
+  end
+
+  def and_I_can_edit_and_reorder_and_delete_steps
+    expect(page).to have_link("Reorder", :href => step_by_step_page_reorder_path(@step_by_step_page))
```

Comment:
Move this to a separate step and make it part of the commit where 'Reorder' is added.